srcpkgs/plex-media-player/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'plex-media-player'
+pkgname=plex-media-player
+version=2.19.0.902
+revision=1
+_commit_rev=42a9f589
+wrksrc="plex-media-player-${version}-${_commit_rev}"
+build_style=cmake
+configure_args="-DFULL_GIT_REVISION=${_commit_rev}"
+hostmakedepends="python3"
+makedepends="libcec-devel libX11-devel libXext-devel libXrandr-devel
+ MesaLib-devel mpv-devel qt5-devel qt5-declarative-devel qt5-location-devel
+ qt5-webchannel-devel qt5-webengine-devel qt5-x11extras-devel SDL2-devel zlib-devel"
+short_desc="Next generation Plex Desktop/Embedded Client"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://github.com/plexinc/plex-media-player"
+distfiles="https://github.com/plexinc/plex-media-player/archive/v${version}-${_commit_rev}.tar.gz"
+checksum=078e4baf8aa5688b021bcb5575cf365e252b1ca79c82737e5391fef01c38d3c9
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-devel qt5-host-tools"
+	configure_args+=" -DQTROOT=/usr/${XBPS_CROSS_TRIPLET}"
```

Comment:
why not just use ${XBPS_CROSS_BASE}/usr

if it isn't cross-compilation XBPS_CROSS_BASE will expand to nothing.

you can even move it to configure_args on top

Reply:
Ah, thanks, that did the trick